src/pretix/base/invoicing/pdf.py

```diff
@@ -752,11 +753,60 @@ def day(dt: datetime.datetime) -> datetime.date:
             return dt.astimezone(tz).date()
 
         total = Decimal('0.00')
+        if has_taxes:
+            colwidths = [a * doc.width for a in (.50, .05, .15, .15, .15)]
+        else:
+            colwidths = [a * doc.width for a in (.65, .20, .15)]
+
         for (description, tax_rate, tax_name, net_value, gross_value, subevent, period_start, period_end), lines in addon_aware_groupby(
             all_lines,
             key=_group_key,
             is_addon=lambda l: l.description.startswith("  +"),
         ):
+            # split description into multiple Paragraphs so each fits in a table cell on a single page
+            # otherwise PDF-build fails
+
+            description_p_list = []
+            # normalize linebreaks to newlines instead of HTML so we can safely substring
+            description = description.replace('<br>', '<br />').replace('<br />\n', '\n').replace('<br />', '\n')
+
+            # start first line with different settings than the rest of the description
+            curr_description = description.split("\n", maxsplit=1)[0]
+            cellpadding = 6  # default cellpadding is only set on right side of column
+            max_width = colwidths[0] - cellpadding
+            max_height = self.stylesheet['Normal'].leading * 5
+            p_style = self.stylesheet['Normal']
+            while True:
```

Comment:
I spent 15min reading the code in this while-loop, and it took me the full 15min to figure out why it is very unlikely that this will ever be an infinite loop (but not completely guaranteed, either).

Infinite loops are a horrible problem to have in production. 

If we go with:

```suggestion
            for __ in range(1000):
```

We have limited the risk, also if it breaks because of a future change that is not done carefully enough